common/app/navigation/helpers/UrlHelpers.scala

```diff
@@ -89,12 +90,12 @@ object UrlHelpers {
       ),
     )
 
-    import com.netaporter.uri.dsl._
+    import io.lemonlabs.uri.typesafe.dsl._
 
     // INTCMP is passed as a separate param because people look at it in Google Analytics
     // It's set to the most specific thing (componentId) to maximise its usefulness
     val url = destination.url ? ("INTCMP" -> componentId) & ("acquisitionData" -> acquisitionData.toString)
-    url.toString(uriEncoder)
+    Url.parse(url.toString).toString
```

Comment:
I think Url.parse can throw an exception (https://github.com/lemonlabsuk/scala-uri/blob/3.x.x/shared/src/test/scala/io/lemonlabs/uri/ParsingTests.scala#L147). I don't _think_ we're introducing a new problem here, just thought I'd flag this.

Reply:
Agreed that, because the exception could've been thrown before this change, we'll leave as-is